pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -413,11 +413,11 @@ var _ = Describe("Schema", func() {
 		It("Generate expected libvirt xml", func() {
 			domain := NewMinimalDomainSpec("mynamespace_testvmi")
 			buf, err := xml.Marshal(domain)
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
 
 			newDomain := DomainSpec{}
 			err = xml.Unmarshal(buf, &newDomain)
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
This fit a direct approach with `.To(Succeed())`

Reply:
Done